packages/opentelemetry-plugin-https/test/utils/Utils.ts

```diff
@@ -0,0 +1,29 @@
+/*!
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as dns from 'dns';
+
+export class Utils {
```

Comment:
I'd prefer to use utility functions over a container class, this is already raised in https://github.com/open-telemetry/opentelemetry-js/issues/345.